### ISSUE TITLE
More information on book updates

### ISSUE THREAD

**Issue by Foxandxss** [teropa/build-your-own-angularjs#38]
Hi Tero,

Your messages with the issues fixes are not clear enough, I mean, you know what you fixed, but we don't. In other words, your messages are really really short so when you fixed some stuff, I need to check all the scope.js file step by step to see what changed.

Add the pages you changed (if you changed code or tests) and where exactly. Sometimes you say, I changed XXX to be more angular-ish, but not sure if it was a bug in the text or you actually modified something in the code.

TL;DR; Make it easy to the early readers to keep track of changes.
